src/index.scss

```diff
@@ -0,0 +1,43 @@
+@import './reset.css';
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.app,
+.button-row,
+.display {
+  display: flex;
+}
+
+.app {
+  width: 700px;
+  flex-direction: column;
+}
+
+.display,
+.button-row {
+  height: 100px;
+}
+
+.display {
+  background-color: #808080;
```

Comment:
- [ ] Let's align the display result to the right

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-alignment.png)

Reply:
Thanks for your feedback. I aligned them to the right by using `justify-content: right.` If it isn't shown as expected in your browser, let me know it, please.

Reply:
Hi @ozovalihasan. I think there is no `justify-content: right`. Please check the available options for [justify-content](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content).
One solution is to have a paragraph tag inside the display div and use `text-align: right` on this paragraph to make it to the right, and without affecting the display div full size.

Reply:
Hi @mosaaleb. When I checked the link you sent, I saw ` justify-content: right` at the end of positional alignments. ![image](https://user-images.githubusercontent.com/44648011/98006138-f38ffd80-1e02-11eb-8910-6614e22031b7.png)
Shouldn't I use it?

Reply:
Sorry, I didn't know that. Definitely, you can use it if it solves the problem.

Reply:
Hi @mosaaleb. I think `justify-content:right` is not working correctly. If you want, you may check [this link](https://github.com/ozovalihasan/react-calculator/pull/3#issuecomment-721690107).